docs/background/disaster-recovery.md

```diff
@@ -0,0 +1,54 @@
+# Disaster recovery
+
+When you go on to create a new server in {{brand}}, you will probably
+notice an option named **Disaster recovery**. That particular option is
+enabled by default.
+
+![Disaster recovery is enabled by default for new
+servers](assets/disaster-recovery-option-on-by-default.png)
+
+Even if you choose to disable it for a particular server, keep in mind
+that you have the option to enable it at a later time.
+
+![The disaster recovery feature can be manually activated for any
+server](assets/disaster-recovery-option-manual-activation.png)
+
+In the following, we explain what this option does, how it works in the
+background, and why you should consider enabling it.
+
+## What it is
+
+The *Disaster Recovery* feature, DR for short, is managed from the
+{{gui}} only and applies to servers and volumes that use our
+[Ceph](https://ceph.io) backend.
+
+## How it works
+
+As soon as you enable DR for a server or a single volume, then you start
+getting snapshots for the corresponding [*RADOS Block Device*
+(RBD)](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Block-Device)
+image. Those snapshots are created automatically once per day, and you
+always have the snapshots of the last 10 days. Please keep in mind that
+all RBD snapshots are created in the same Ceph cluster and are not
+replicated remotely. If, for any reason, you delete the original volume,
+then all snapshots will also be deleted, and the snapshot creation
+schedule will be canceled immediately.
+
+## Why enable it
+
+Provided snapshots are available, you can restore a server or a single
+volume to any of those snapshots. For instance, you may discover that
+due to faulty application logic or simply a bug, you are now
+experiencing data corruption. Then, one of your options would be to go
+back in time by restoring one of the available snapshots and keep going
```

Comment:
This paragraph should contain a link to a short how-to guide for resetting a volume or server to the most recent snapshot state.

Reply:
I'd say we want to roll that into this same PR. Information on backups/snapshots is mildly useless unless accompanied by information on how to restore. :)

Reply:
...aaaaand I rolled that in :)

Reply:
I meant roll it into the same *PR*, not roll it into the same file. We want a *separate* how-to guide, *as part of this PR,* that cross-references the background info (and vice versa). The step-by-step guide for restoring doesn't belong in background information.

Reply:
Ahem, rollin' in and updating --- but properly, this time aroond!